content/planning-framework/overview.mdx

```diff
@@ -1,8 +1,49 @@
 ---
 id: 'planning-framework-overview'
 title: 'Planning Framework'
-subtitle: ''
+subtitle: 'Surfacing key questions to shape scientific software development'
 ---
-# Overview
+import { Grid, StepLabel, StepContent } from '@mui/material';
+import { Hero } from '/src/components/Hero';
+import { PageContainer } from '/src/components/PageContainer';
+import { ContentCard } from '/src/components/ContentCard';
+import { ContentStepper, ContentStep } from '/src/components/ContentStepper';
+import { Link } from '@mui/material';
 
-Labore mollit incididunt irure occaecat nulla officia sint nulla sint nulla. Officia irure aliquip veniam culpa Lorem Lorem ullamco pariatur pariatur. Anim eiusmod eiusmod duis adipisicing anim adipisicing officia. Do est adipisicing ullamco sint excepteur quis.
\ No newline at end of file
+<Hero>
+  The STRUDEL Planning Framework will help scientific teams incorporate UX practices in their software production.
+
+  ## The STRUDEL Planning Framework
+
+  <Grid container spacing={4}>
+    <Grid item md={4}>
+      <ContentCard>
+        #### Surfaces key questions
+
+        Surfaces key questions to consider when planning & designing new software products or features
+      </ContentCard>
+    </Grid>
+    <Grid item md={4}>
+      <ContentCard>
+        #### Foregrounds sociotechnical matters
+
+        Consider organizational, social, & technical topics affecting team work
+      </ContentCard>
+    </Grid>
+    <Grid item md={4}>
+      <ContentCard>
+        #### Informed by typology research
+
+        Key questions surfaced through research & development of sociotechnical typology
+      </ContentCard>
+    </Grid>
+  </Grid>
+</Hero>
+
+<PageContainer>
+# Learn more
+More information is coming soon!
+
+Understand the background by reviewing our initial <Link href="/planning-framework/typology">typology</Link>.
```

Comment:
One issue I'm noticing with switching to the gatsby instead of mui for Link is that the link itself is not underlined and therefore invisible on the page. Is there a styling setting elsewhere that you need to change @codytodonnell?